sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <v-table class="bordered">
+    <thead>
+      <tr>
+        <th scope="col" class="text-left">Loan/grant type</th>
```

Comment:
why `scope="col"`?

Reply:
It is from sonar rule to use th always scope or id.

https://rules.sonarsource.com/html/RSPEC-1102